src/test/java/io/github/thebusybiscuit/slimefun4/core/guide/TestGuideOpening.java

```diff
@@ -90,8 +93,28 @@ void testOpenItemStack() throws InterruptedException {
     }
 
     @Test
-    @DisplayName("Test if the Slimefun Search can be opened from the History")
+    @DisplayName("Test if the Slimefun Search works with normal and colored querys")
     void testOpenSearch() throws InterruptedException {
+        String normalQuery = "iron";
+        String coloredQuery = ChatColor.DARK_PURPLE + "iron";
+
+        SlimefunItem testItem = TestUtilities.mockSlimefunItem(plugin, "IRON_ITEM", new CustomItemStack(Material.IRON_INGOT, "iron item"));
+        testItem.register(plugin);
+
+        Player player = server.addPlayer();
+        PlayerProfile profile = TestUtilities.awaitProfile(player);
+        SlimefunGuideImplementation guide = new SurvivalSlimefunGuide(false, false);
+
+        guide.openSearch(profile, normalQuery, false);
+        Assertions.assertTrue(player.getOpenInventory().getTopInventory().contains(testItem.getItem()), "Failed on normal query");
+
+        guide.openSearch(profile, coloredQuery, false);
+        Assertions.assertTrue(player.getOpenInventory().getTopInventory().contains(testItem.getItem()), "Failed on colored query");
+    }
+
+    @Test
+    @DisplayName("Test if the Slimefun Search can be opened from the History")
+    void testOpenSearchHistory() throws InterruptedException {
```

Comment:
this isn't testing history, it's just a basic search term open

I'd rename this back

Reply:
Then what does the display name mean?

Reply:
Nah the mockito call doesn't do anything for that but I missed that it did a history.add in the playerprofile setup

So yeah we can keep the name